### PR TITLE
File structure fix

### PR DESCRIPTION
**Pull request by mki98**
## Is this issue already raised? Yes/No
No
## Topic: 
The structure of the files contained duplicate files and the layout wasn't clear. I made a simple change to the layout which is more clear

### CODE REVIEW

**Review by ishtms — COMMENTED** [review]
@mki98  Thanks for the suggestion. But doesn't the current file structure suggest that - `config/` folder and the files - `index.js`, `package.json` and `logger.js` live in the same directory?

File: chapters/ch04.1-refactoring-the-code.md [hunk 1]
```diff
@@ -39,9 +39,8 @@ lib/
 ├── config/
 │     ├── rolling-config.js
 │     ├── log-config.js
-├── index.js (entry point)
-├── package.json
-└── logger.js
```
Comment (ishtms) [inline chapters/ch04.1-refactoring-the-code.md:44]: We do need the `logger.js` file here.

**Review by ishtms — COMMENTED** [review]
This is also needed, I think.

File: chapters/ch04.1-refactoring-the-code.md (hunk 1 above)
Comment (mki98) [reply chapters/ch04.1-refactoring-the-code.md:44]: **First I would like to thank you so much for your incredible work on this book <3**




Correct me if I am wrong, the following screenshot has no` logger.js` in the root but the structure shows that it is in the root.

image

**This following screenshot shows that it is in the `lib/` directory not the root**   

image

Comment (ishtms) [reply chapters/ch04.1-refactoring-the-code.md:44]: Ah! you're totally correct, my bad. That in-fact is a wrong directory structure that I've posted in the chapter 4.1
Thanks for clearing it up @mki98, cheers!

**Review by ishtms — APPROVED** [review]
Makes sense now.

File: chapters/ch04.1-refactoring-the-code.md (hunk 1 above)
Comment (mki98) [reply chapters/ch04.1-refactoring-the-code.md:44]: You're welcome! I'm glad I could help ❤️

**Pull request merged**